### ISSUE TITLE
Mailing list link should be very easy to find

### ISSUE THREAD

**Issue by fperez** [jupyter/jupyter.github.io#45]
As far as I can tell, it's nowhere on the front page, it should be prominently linked in more than one place so that users always find it (like it was in the old ipython site)

**Comment by minrk** [comment]
cc @cameronoelsen 

perhaps a contact page, with various links?

**Comment by ellisonbg** [comment]
@willingc and I had talked about a "Community" page in the docs or website that has this and other community related things (ipython-dev, stackoverflow, gitter, etc.)

Not sure if docs or website makes more sense. We could always link from the website to the docs like we do with install.

**Comment by willingc** [comment]
We're talking about the jupyter@googlegroups.com email. Correct?

Assuming that's the case, adding to the footer of the webpage with the other social media buttons would be helpful. Perhaps also somewhere on the Project/About page and the Donate page.

We could certainly add to the docs. Also perhaps a badge on each repo's README.

**Comment by willingc** [comment]
@minrk's suggestion of a contact page is also a good one.

**Comment by fperez** [comment]
Correct, @willingc. Indeed a community/contact page is a good idea, but it should also be listed on the main page. This information should be *really easy* to find, in more than one place.

**Comment by fperez** [comment]
Actually @willingc let me rephrase, I meant the [google group page](https://groups.google.com/forum/#!forum/jupyter), so folks can decide whether to join via email, the web interface, etc.  I think that's better than providing directly the mail-to address. Sorry about my previous response, I misread your question.

**Comment by cameronoelsen** [comment]
If we want to make a page on the website for this, I could probably flesh out something in a week or two if we have all the content that we want on the page!

**Comment by cameronoelsen** [comment]
Since we now have a community page that contains all of the information on our mailing lists as well as our other communities (Gitter, StackOverflow), I am going to close this issue
